tests/ha/4gws_create_delete.sh

```diff
@@ -156,10 +156,11 @@ for i in $(seq 0 $(expr $FAILING_GATEWAYS - 1)); do
   echo  📫 nvme-gw delete gateway: \'$gw_name\' pool: \'$POOL\', group: \'\' \(empty string\)
   docker compose exec -T ceph ceph nvme-gw delete $gw_name $POOL ''
 done
-
+sleep 100 # wait for scale down rebalance complete
 docker ps
 
-# expect remaining gws to have two optimized groups each
```

Comment:
Great job! Perhaps it would be worth adding a full state transition test. I'm not entirely sure of the exact sequence, but it seems like it might be: (all available) → (gateway delete) → (failover for 4 ANA groups) → (rebalance 2 ANA groups). Alternatively, could the rebalance potentially finish before the failover? What do you think?

Reply:
In this PR I slightly modified your test 4GW_create_delete - when you delete 2 GW I added delay sufficient for  rebalance process to redestribute ana groups to remaining 2 GW. when last namespace leaves ANA group monitor removes gw in DELETING state from database also ANA group is removed. We now test this scenario in this test

Also I added test called auto_load_balance. it in loop : deletes GW - wait for re-balance- calculates number namespaces in each ana group  per each subsystem - creates GW - wait for re-balance - calculates the same..

Maybe we can later add another test  with 4 GWs: delete one or two GWs and during  scale up re-balance  simulate the crash of one of remaining gws. Then create back 2 GWs and during scale-up re-balance  again simulate GW stop/ start